CurrencyConverterService/CurrencyRefreshing.cs

```diff
@@ -0,0 +1,51 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using Microsoft.EntityFrameworkCore;
+using CurrencyConverterService.Models;
+using System.Linq;
+
+namespace CurrencyConverterService
+{
+    class CurrencyRefreshing
+    {
+        Context dataBase;
+        Logging logging;
+        public CurrencyRefreshing()
+        {
+            this.dataBase = new Context();
```

Comment:
Use Dependency Injection to make this class testable at least. Example https://www.codeproject.com/Tips/794436/Constructor-Dependency-Injection-Pattern-Implement